packages/@react-aria/tooltip/src/useTooltip.ts

```diff
@@ -25,14 +25,13 @@ export function useTooltip(props: SpectrumTooltipProps): TooltipAria {
     role = 'tooltip'
   } = props;
 
-  let {onPointerLeave, onPointerEnter} = useContext(DOMPropsResponderContext) || {};
+  let focusableProps = useContext(FocusableContext);
```

Comment:
minor thing, but perhaps you could use `useFocusableContext()` instead of `useContext` here?

Reply:
useFocusableContext requires ref as argument (useTooltip does not have ref)... other way is to make ref as optional argument in useFocusableContext, if thats ok?